src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedFieldSymbol.cs

```diff
@@ -85,6 +85,8 @@ internal override bool IsNotSerialized
             }
         }
 
+        internal override bool HasPointerType => _underlyingField.HasPointerType;
```

Comment:
See `GenericConstraintsTests.UnmanagedRecursiveGenericStruct` to see the significance of this change. This prevents us from binding recursively until we blow the stack when we check if those types meet their managed constraints.